PayKids/presentation/src/main/java/com/paykids/presentation/view/quiz/StudyFragment.kt

```
@@ -110,6 +120,20 @@ class StudyFragment : BaseFragment<FragmentStudyBinding>() {
         binding.etSendChat.text.clear()
     }
 
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        backPressedCallback = object : OnBackPressedCallback(true) {
+            override fun handleOnBackPressed() {
+            }
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)